cli/pcluster/commands.py

```diff
@@ -706,7 +695,18 @@ def command(args, extra_args):  # noqa: C901 FIXME!!!
 
         # run command
         if not args.dryrun:
-            os.system(cmd)
+            if return_output:
+                try:
+                    return sub.check_output(cmd, shell=True, stderr=sub.STDOUT)
+                except sub.CalledProcessError as e:
+                    error = e.output.decode("utf-8")
+                    if "/opt/parallelcluster/scripts/pcluster_dcv_connect.sh: No such file or directory" in error:
+                        LOGGER.error("The version of ParallelCluster in the machine does not support DCV")
```

Comment:
We can improve the message with something like "The cluster "..." has been created with an old version of ParallelCluster without the DCV support."

Reply:
Fixed in the # 1314